src/GafferUIBindings/PlugAdderBinding.cpp

```diff
@@ -89,13 +92,71 @@ struct PlugMenuSlotCaller
 
 };
 
+struct PlugAdderWrapper : public PlugAdder, IECorePython::Wrapper< PlugAdder >
+{
+
+	PlugAdderWrapper(PyObject *self, StandardNodeGadget::Edge edge ) : PlugAdder( edge ), IECorePython::Wrapper<PlugAdder>( self, this )
+	{
+	}
+
+	bool acceptsPlug( const Gaffer::Plug *connectionEndPoint ) const
+	{
+		IECorePython::ScopedGILLock gilLock;
+		override o = this->get_override( "acceptsPlug" );
+		if( o )
+		{
+			try
+			{
+				return o( Gaffer::PlugPtr( const_cast<Gaffer::Plug *>( connectionEndPoint ) ) );
+			}
+			catch ( error_already_set )
+			{
+				PyErr_Print();
```

Comment:
Use `ExceptionAlgo::translatePythonException()` to properly convert the exception to C++ and propagate it up the call stack. Here and there we still have old code that buries exceptions with `PyErr_Print()`, but over time I'm moving them over to using proper handling.